rules/TypeDeclaration/Rector/Class_/TypedPropertyFromCreateMockAssignRector.php

```diff
@@ -84,11 +84,6 @@ public function refactor(Node $node): ?Node
         $hasChanged = false;
 
         foreach ($node->getProperties() as $property) {
-            // already typed
-            if ($property->type instanceof Node) {
```

Comment:
I think this should check if `property->type` already `PHPUnit\Framework\MockObject\MockObject`, then continue to next property, to avoid repetitive apply

I will create separate PR.